src/systemtest/java/com/ericsson/ei/systemtest/utils/Config.java

```diff
@@ -37,11 +38,13 @@ public Config() {
     * @return
     */
     public void initJenkinsConfig() {
-        jenkinsBaseUrl = System.getProperty("jenkins.url");
+        jenkinsExternalBaseUrl = System.getProperty("jenkins.external.url");
```

Comment:
Maybe add a comment to these variables for future references? To explain when the external vs the internal URL are used.